cypress/e2e/contacts.cy.js

```diff
@@ -46,8 +46,58 @@ describe("Contacts", () => {
+            cy.get(".govuk-error-summary__body").should("contain", "Enter an email address in the correct format, like name@example.com");
+            cy.get(".govuk-error-summary__body").should("contain", "The telephone number must be 255 characters or fewer");
+            cy.get(".govuk-error-summary__body").should("contain", "The other telephone number must be 255 characters or fewer");
+            cy.get(".govuk-error-summary__body").should("contain", "The note must be 255 characters or fewer");
+        });
+    });
+
+    describe("Updating a Contact", () => {
+        beforeEach(() => {
+            cy.visit("/supervision/deputies/3/contacts");
+            cy.get(':nth-child(2) > :nth-child(4) > .govuk-button--secondary').click();
+        })
+
+        it("shows content", () => {
+            cy.get(".govuk-main-wrapper > header").contains("Manage contact");
+            cy.get("#contact-form > :nth-child(2) > .govuk-label").contains("Name (required)");
+            cy.get("#contact-form > :nth-child(3) > .govuk-label").contains("Job title");
+            cy.get("#contact-form > :nth-child(4) > .govuk-label").contains("Email (required)");
+            cy.get("#contact-form > :nth-child(5) > .govuk-label").contains("Phone (required)");
+            cy.get("#contact-form > :nth-child(6) > .govuk-label").contains("Other phone");
+            cy.get("#f-contact-notes > .govuk-label").contains("Notes");
+            cy.get("#f-isNamedDeputy > .govuk-fieldset__legend").contains("Named deputy (required)");
+            cy.get("#f-isMainContact > .govuk-fieldset__legend").contains("Main contact (required)");
+            cy.get(".govuk-button").contains("Save contact");
+            cy.get(".govuk-button-group > .govuk-link").contains("Cancel");
+        });
+
+        it("should allow me to submit the form", () => {
+            cy.setCookie("success-route", "updateContact");
+            cy.get("#f-contactName").type("{selectAll}{backspace}John Smith");
+            cy.get("#contact-form").submit();
+            cy.url().should("contain", "/supervision/deputies/3/contacts?success=updatedContact");
+            cy.get(".moj-banner").should("contain", "John Smith's details updated");
+        });
+
+        it("shows error message when submitting empty data", () => {
+            cy.setCookie("fail-route", "manageContactEmpty");
+            cy.get("#contact-form").submit();
+            cy.get(".govuk-error-summary__body").should("contain", "Enter a name");
+            cy.get(".govuk-error-summary__body").should("contain", "Enter a telephone number");
+            cy.get(".govuk-error-summary__body").should("contain", "Enter an email address");
+            cy.get(".govuk-error-summary__body").should("contain", "Select whether this contact is a main contact");
+            cy.get(".govuk-error-summary__body").should("contain", "Select whether this contact is the named deputy");
```

Comment:
If you are using should, you shouldn't have to keep "get"ting the element again. I think you can just chain them, e.g. `cy.get("...").should("contain", "a").should("contain", "b")...